test/Network/Packet.test.cpp

```diff
@@ -239,7 +239,7 @@ TEST_CASE("[Network] sf::Packet")
 
         SECTION("char*")
         {
-            const char string[] = "testing";
+            const char string[] = "testing"; // NOLINT(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays)
```

Comment:
It’s kind of funny how the section uses `char*`, but then in the code we use `const char string[]`. Given that we test C-style strings here, which are usually written as `char*`, wouldn’t it make sense to just use `const char* string`?